src/Error/ErrorLogger.php

```diff
@@ -65,6 +65,11 @@ public function logMessage($level, string $message, array $context = []): bool
         if (!empty($context['trace'])) {
             $message .= "\nTrace:\n" . $context['trace'] . "\n";
         }
+        $logMap = [
+            'strict' => LOG_NOTICE,
+            'deprecated' => LOG_NOTICE,
+        ];
```

Comment:
Seems odd that it's just these 2 levels here.

Reply:
The other levels don't need to be mapped as they are already correct.